edb/schema/referencing.py

```diff
@@ -181,20 +200,30 @@ def get_implicit_bases(self, schema):
 
 class ReferencedObjectCommandMeta(sd.ObjectCommandMeta):
     _transparent_adapter_subclass: ClassVar[bool] = True
-
-    def __new__(mcls, name, bases, clsdct, *,
-                referrer_context_class=None, **kwargs):
+    _referrer_context_class: Optional[
```

Comment:
This is a class attribute, so a `ClassVar` is needed.

Reply:
I just tried to add `ClassVar`, a new error is reported by mypy when this property is set inside the `__new__` method, close to its definition.
`Cannot assign to class variable "_referrer_context_class" via instance`.

However, since `__new__` there belongs to a metaclass, shouldn't mypy tolerate setting a class attribute here?

Reply:
Ah, sorry, I missed that this is a metaclass. No `ClassVar` then.